lib/internal/policy/manifest.js

```diff
@@ -354,6 +345,24 @@ function findScopeHREF(href, scopeStore, allowSame) {
  * @typedef {boolean | string | SRI[] | typeof kCascade} Integrity
  */
 
+let URL;
+let shouldAbortOnUncaughtException;
+let createHash, timingSafeEqual, HashUpdate, HashDigest;
+// To get this in the snapshot we can't have things that use getOptionsValue or
+// expose C++ functions so we do these lazily
```

Comment:
It should be possible to include C++ bindings in the snapshot though - are you having trouble with crypto? We'd need to make sure that InitCryptoOnce() is called again during deserialization if crypto is included (see my TODO in node_crypto.cc)

Reply:
crypto and URL both give a Trap code 5 (I couldn't figure out what that means exactly) and an `Unresolved` line in the STDERR during mksnapshot. This makes it work I'm just trying to cleanup policy for now and not sure of cleaning up crypto and how to make it work with serialization in a short timeline.

Reply:
I am guessing you are running into unregistered external reference errors (check out [the src/README.md](https://github.com/nodejs/node/tree/master/src#binding-functions) for explanations and how to deal with this). Anyway I opened https://github.com/nodejs/node/pull/42203 to support crypto in the bootstrap snapshot - not a blocker for this PR, though, I am okay with landing this while still lazy loading crypto.

Reply:
rebased and cleaner for sure